src/Components/ListenOn/ListenOn.js

```diff
@@ -46,7 +46,7 @@ export default function ListenOn(props) {
 
                 {listenLinks}
             </div>
-            <Link className="navLink" to="/About">
```

Comment:
if you no longer need this class you can also remove any stlying that went with it as well